app/controllers/miq_ae_class_controller.rb

```diff
@@ -1815,6 +1816,31 @@ def namespace
     render :json => find_record_with_rbac(MiqAeNamespace, params[:id]).attributes.slice('name', 'description', 'enabled')
   end
 
+  def ae_domains
+    domains = MiqAeDomain.where("ancestry is null and enabled = ?", true).order("name").select("id, name")
+    render :json => {:domains => domains}
+  end
+
+  def ae_methods
+    methods = MiqAeMethod
+              .name_path_search(params[:search])
+              .domain_search(params[:domain_id])
+              .selected_methods(params[:ids])
```

Comment:
```suggestion
              .selected_methods(params[:ids]&.split(',').&map(&:to_i)
```

Though I don't think the `to_i` is necessary

Reply:
redacted